modules/ROOT/pages/clauses/create.adoc

```diff
@@ -205,6 +205,57 @@ Nodes created: 2 +
 Properties set: 4
 |===
 
+[role=label--new-5.26]
+[[dynamic-create]]
+== CREATE nodes and relationships using dynamic node labels and relationship types
+
+Node labels and relationship types can be referenced dynamically in parameters and variables when creating nodes and relationships.
```

Comment:
Should we say they can be dynamically calculated from expressions? Because it isn't only parameters and variables that are allowed